MoA/models/llama/modeling_llama.py

```diff
@@ -54,33 +54,24 @@ def LlamaModel_MixtureAttention_forward(
     input_ids: torch.LongTensor = None,
     attention_mask: Optional[torch.Tensor] = None,
     position_ids: Optional[torch.LongTensor] = None,
-    past_key_values: Optional[List[torch.FloatTensor]] = None,
+    past_key_values: Optional[Union[Cache, List[torch.FloatTensor]]] = None,
```

Comment:
Perhaps Cache should be replaced as StaticCircularCache? Since this is the forward function of MoA and the default cache is not the regular cache. We may want to highlight that.

Reply:
You are right. I replace `Cache` with `StaticCircularCache` in next commit.